ha/virsh/tests/resource_lvm-activate_test.sh

```diff
@@ -66,9 +66,9 @@ test_move_resource() {
   find_node_to_move_resource "${RESOURCE_NAME}"
 
   # Move resource to another VM
-  run_command_in_node "${IP_VM01}" "sudo crm resource move ${RESOURCE_NAME} ${VM_TARGET}"
+  run_command_in_node "${IP_VM01}" "sudo pcs resource move ${RESOURCE_NAME} ${VM_TARGET}"
   sleep 3
```

Comment:
Could we also use `--wait` here instead of a hardcoded  `sleep`? It sounds like the `move` command would then block until the operation is finished, or the `wait` period is over, with a proper non-zero exit status if it failed (or hit the wait timeout I suppose).